src/Atmos/Model/tendencies_energy.jl

```diff
@@ -26,6 +26,16 @@ function flux(::ViscousFlux{Energy}, atmos, args)
     return τ * state.ρu
 end
 
+function flux(::HyperdiffViscousFlux{Energy}, atmos, args)
+    @unpack state, hyperdiffusive = args
+    return hyperdiffusive.hyperdiffusion.ν∇³u_h * state.ρu
```

Comment:
Maybe some different names could be used here? `hyperdiffusive.hyperdiffusion` seems a little too hyper?

Reply:
I agree that it's a bit hyper, but it might be nice to change the name across the board since `hyperdiffusive.hyperdiffusion` exists in a bunch of other places. Maybe in a follow-up PR?

Reply:
sounds good